spikeinterface/sortingcomponents/waveforms/waveform_utils.py

```diff
@@ -0,0 +1,20 @@
+def to_channelless_representation(waveforms):
```

Comment:
As discussed in the meeting, I think this is not specific enough (stack how? unstack from where?). I think that is better to name operations that reflect the domain of application. If, temporal representation is more meaningful term in this domain I think that `to_temporal_representation` is better (that is, if it is obvious for everyone that this means "throw the spatial information out and consider every channel an independent sample"). Moreover, this can be explained in the docstring and this is why it is useful to have a common method for the scattered use of this operation.

I think that "channeless_representation" point specifically to what the method is doing but is not a common word in the English language ( google n-grams reports a strange usage pattern)

Reply:
Ok for me!